styleguide.html

```diff
@@ -5,15 +5,7 @@
 <title>Styleguide - Cafébebel 2017</title>
 <link rel=stylesheet href=./styleguide.css>
 
-<h1>
-  <a href=#>
-  <span class=icon-logo-cafebabel>
-    <span class=path1></span>
-    <span class=path2></span>
-    <span class="text-hidden">Cafébabel</span>
-  </span>
-  </a>
-</h1>
+<a href="#" class=logo-cafebabel><img src=./img/logo-cafebabel.svg alt="logo cafébabel"></a>
```

Comment:
href=#
is the `class=logo-cafebabel` mandatory?

Reply:
well how do you target the element otherwise ? there will probably be a lot of `<img>` tag inside `<a>` tag in the page, right ?